contracts/test/utils/StdLib.sol

```diff
@@ -0,0 +1,306 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./Vm.sol";
+
+// Wrappers around Cheatcodes to avoid footguns
+abstract contract StdLib {
```

Comment:
where is this lib coming from ?

Reply:
Was in a `forge` example repo. Contains lots of useful wrappers such as `deployCode()`, which makes deploying contracts compiled with a different version of straightforward